README.md

```diff
@@ -81,7 +81,7 @@ Latest installers with Mamba and PyPy in the base environment:
 
 ## Install
 
-### Unix-like platforms
+### Unix-like platforms (Mac OS & Linux)
```

Comment:
+1 to this if y'all are open to the fix. it's confusing for new users to know that Unix like means mac! trying to help people install these smaller light weight python distros!!